harness/byteConversionValues.js

```diff
@@ -139,7 +169,22 @@ var byteConversionValues = {
       0,   // -4294967296
       0,   // Infinity
       0,   // -Infinity
-      0
+      0,   // 0
+      1,   // 2049
+      3,   // 2051
+      0,   // 0.00006103515625
+      0,   // 0.00006097555160522461
+      0,   // 5.960464477539063e-8
+      0,   // 2.9802322387695312e-8
+      0,   // 2.980232238769532e-8
+      0,   // 8.940696716308594e-8
+      0,   // 1.4901161193847656e-7
+      0,   // 1.490116119384766e-7
+      224, // 65504
+      240, // 65520
+      239, // 65519.99999999999
+      0,   // 0.000061005353927612305
+      0    // 0.0000610053539276123
```

Comment:
I didn't check how this is specified, but presumably these are equal to truncate(value) modulo 256, with the appropriate sign modifications for signed int8?